daemon/execdriver/native/create.go

```diff
@@ -201,6 +205,18 @@ func (d *driver) setupCgroups(container *libcontainer.Config, c *execdriver.Comm
 	return nil
 }
 
+func (d *driver) setupRlimits(container *libcontainer.Config, c *execdriver.Command) error {
+	if c.Resources == nil {
+		return nil
+	}
+
+	for _, rlimit := range c.Resources.Rlimits {
+		container.Rlimits = append(container.Rlimits, libcontainer.Rlimit((*rlimit)))
+	}
+
+	return nil
```

Comment:
Do we really need an `error` return? It's always `nil` here, not sure about other drivers.